src/components/atoms/DatePicker.vue

```diff
@@ -0,0 +1,57 @@
+<template>
```

Comment:
While I like how the component looks and works I revcieve many warnings in the browser console while using it, double check on your side if you don't have them as well.

Reply:
Fixed, was using timestamp instead of Date/string.